Certification standard/main.html

```diff
@@ -0,0 +1,221 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title>Cardano Certification Level Standards</title>
+    <script src='./respec-ccwg-commons.js' async class='remove'></script>
+  </head>
+
+  <body>
+    <section id='abstract'> 
+
+        This document defines the specification for certification of smart contracts on the Cardano blockchain. It outlines three different levels of certification which can be used to show that a smart contract went has been reviewed and was found to behave as expected and to not be vulnerable to a set of security weaknesses. 
```

Comment:
Typically smart contracts are reviewed in sets, as the set of contracts constituting a DApp, rather than individually. And indeed that's stated on line 54 below.